src/features/hotspots/root/HotspotsView.tsx

```diff
@@ -407,6 +407,7 @@ const HotspotsView = ({
           hotspot={selectedHotspot}
           onLayoutHeader={handleDetailHeaderLayout}
           onFailure={handleBack}
+          onSelectHotspot={handlePresentDetails()}
```

Comment:
I'm not sure why this function is curried? Could just be a carryover from a change that was made. Doesn't hurt anything, but could be cleaned up.

Reply:
Updated it